main.go

```diff
@@ -804,7 +804,7 @@ func main() {
 		fmt.Printf(serviceInfo())
 		addr := fmt.Sprintf(":%d", *port)
 		http.HandleFunc("/", welcome)
-		http.ListenAndServe(addr, http.HandlerFunc(router))
+		log.Fatal(http.ListenAndServe(addr, http.HandlerFunc(router)))
```

Comment:
Pulled what I had originally done from the docs. Routed the request using the logger. That should be the first fix then we can get fancy. See updated commit in pull request to validate changes.